CONTRIBUTING.md

```diff
@@ -53,21 +55,12 @@ Each component requires the following items in order to be complete:
 - A **component class** using vanilla JS + SCSS
 - A **README.md** in its subdir which contains developer documentation on the component, including usage.
 - A **set of unit tests** within `test/unit/` with adequate coverage (which we enforce automatically).
-- A **demo page** within `demos/` that shows example usage of the component.
+- A **screenshot test** within `test/screenshot/spec/` that shows example usage of the component. For more on screenshot tests, please refer to the [screenshot testing documentation](./test/screenshot/README.md).
 
 You can find much more information with respect to building components within our [authoring components guide](./docs/authoring-components.md)
 
 ### Running development server
 
-#### Local development server
```

Comment:
Instead of leaving this section blank, we can suggest using `npm run start` as development environment.